recipe/conda_build_config.yaml

```diff
@@ -306,8 +306,6 @@ pin_run_as_build:
     max_pin: x.x
   tiledb:
     max_pin: x.x.x
-  vc:
-    max_pin: x
```

Comment:
@isuruf I removed this max pin since it appears to be wrong and covered by the run exports.

Reply:
Yes, it's covered by run exports. Btw, `x` is correct, right?

Reply:
I always get confused by this syntax. We set max_pin=None in the run exports, not 'x' IIRC. Doesn't that mean no entry here?

Reply:
https://github.com/conda-forge/vc-feedstock/blob/master/recipe/meta.yaml#L34

Reply:
We set `max_pin=x` at https://github.com/conda-forge/vc-feedstock/blob/master/recipe/meta.yaml#L33

Reply:
Ack wrong line. Yes 'x' is right.